build.gradle

```diff
@@ -6,6 +6,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:2.0.0-beta6'
```

Comment:
These should go in the Android project's Gradle config as they aren't used anywhere else.